lib/modules/platform/github/index.ts

```diff
@@ -1171,23 +1170,22 @@ export async function setBranchStatus({
 
 /* istanbul ignore next */
 async function getIssues(): Promise<Issue[]> {
-  const result = await githubApi.queryRepoField<Issue>(
-    getIssuesQuery,
-    'issues',
-    {
-      variables: {
-        owner: config.repositoryOwner,
-        name: config.repositoryName,
-        user: config.renovateUsername,
-      },
-    },
-  );
-
-  logger.debug(`Retrieved ${result.length} issues`);
-  return result.map((issue) => ({
-    ...issue,
+  const result: Issue[] = (
+    await githubApi.getJson<Issue[]>(
+      `repos/${config.parentRepo ?? config.repository}/issues?creator=${
+        config.renovateUsername
+      }&state=all`,
+      { repoCache: true },
```

Comment:
don't we need to set paginate to true?
I think we only get one page otherwise?
there can be more issues if this is set:

https://docs.renovatebot.com/configuration-options/#configwarningreuseissue

Reply:
I think fetching the last 100 issues will be ok 100% of the time.. but I'll look into it